packages/context/src/__tests__/unit/context-observer.unit.ts

```diff
@@ -270,6 +268,30 @@ describe('Context', () => {
       expect(await getControllers()).to.eql(['3']);
     });
 
+    it('reports error on current context if an observer fails', async () => {
+      const err = new Error('something wrong');
+      server.subscribe((event, binding) => {
+        if (binding.key === 'bar') {
+          return Promise.reject(err);
+        }
+      });
+      server.bind('bar').to('bar-value');
+      const obj = await pEvent(server, 'error');
+      expect(obj).to.equal(err);
+    });
+
+    it('reports error on parent context if an observer fails', async () => {
```

Comment:
I see two test cases with different observed behavior here, but the test names don't explain why is the behavior different.
- it reports error on current context if an observer fails
- reports error on parent context if an observer fails

I am proposing to rename this test case to something like:

```ts
it('reports error on parent context when there are no error listeners');
```